peer/brontide_test.go

```diff
@@ -993,10 +993,10 @@ func genScript(t *testing.T, address string) lnwire.DeliveryAddress {
 	return script
 }
 
-// TestPeerCustomMessage tests custom message exchange between peers.
-func TestPeerCustomMessage(t *testing.T) {
-	t.Parallel()
-
+// SetUpTest sets up a test environment for the custom
+// message exchange between peers.
+func SetUpTest(t *testing.T) ([33]byte, *Brontide, chan *customMsg,
```

Comment:
For now, I don't think this function should be created. Instead, `TestPeerUnknownMessageOdd/Even` can copy what they need and use the table-driven golang paradigm. So maybe a `TestPeerUnknownMessage` with a case for even and another one for odd.

Reply:
Yeap it's just that `TestPeerCustomMessage` and `TestPeerUnknownMessage` need the same set up code, so felt like a lot of code duplication.